docs/reference/server-configuration.md

```diff
@@ -734,9 +734,9 @@ For more information, see [Processing Guarantees](/operate-and-deploy/exactly-on
 **Per query:** no (may be set with ALTER SYSTEM, for {{ site.ccloud }} only)
 
 The type of compression used by streams producers for topics created by INSERT INTO, 
-CREATE TABLE AS SELECT, and CREATE STREAM AS SELECT statements. The default is `snappy`.
+CREATE TABLE AS SELECT, and CREATE STREAM AS SELECT statements. The default is `none`.
 
-This setting is distinct from the `ksql.compression.type` config, which sets the
+This setting is distinct from the `compression.type` config, which sets the
```

Comment:
Ooo.... I didn't see these two lines before. 

I am unaware of how this works.  We should find someone else to chime on this change!  (Sorry to be unhelpful, Jim!)

Reply:
kk cool, @jnh5y  -- @colinhicks @suhas-satish how do think we should proceed?